test/m1_facilitator/SeedDataInitializer/initialize.test.ts

```diff
@@ -263,4 +274,83 @@ describe('SeedDataInitializer:initialize', () => {
       auxiliaryAnchorAddress,
     );
   });
+
+  it('should initialize seed data when for supporting older testnet', async (): Promise<void> => {
```

Comment:
What exactly is `older` testnet? 🤔   After few months `Gen1` testnets will also be old.

Reply:
Changed it to : `should initialize seed data when facilitator is running on GEN0 testnet`